app/controllers/news_items_controller.rb

```diff
@@ -124,6 +124,11 @@ def edit
   end
 
   def create
+    unless category_present?(params.require(:news_item).permit!)
+      flash[:error] = "Bitte eine Kategorie auswählen"
+      redirect_to new_news_item_path and return
+    end
+
     query = create_params
     begin
```

Comment:
Entweder bitte `params.require(:news_item).permit!` als eigene Methode definieren:

```
def news_item_params
  params.require(:news_item).permit!
end
```

oder direct in der Methode `def category_present?(params)` die Params Umschreiben, wenn sie sonst nicht weiter verwendet werden.

Reply:
done in b815823